### DISCUSSION TITLE
Checking if an inequality is Boolean

### DISCUSSION THREAD

**Discussion by Adar70** [sympy/sympy#26768] [General]
When I'm checking if an inequality is Boolean using isinstance(), it always gives me True. Here's minimal example:
`a = (x + 2 > 3)

a
Out[52]: x + 2 > 3

isinstance(a, Boolean)
Out[53]: True`

**Comment by oscarbenjamin** [comment]
I'm converting this into a discussion because it is not an issue with sympy.

I don't know what you expect to happen but it is correct that `isinstance(a, Boolean)` should always be true.

**Comment by Adar70** [comment]
I'm expecting that I get false from `isinstance(a, Boolean)` because it's neither `sympy.true` nor `sympy.false`. It's a relational expression.

**Comment by asmeurer** [comment]
Boolean in SymPy is the superclass for any sort of symbolic boolean expression. To see if something evaluates to True or False, you can just check `if expr in [True, False]`.
